app/views/state_file/questions/return_status/edit.html.erb

```diff
@@ -2,7 +2,7 @@
 <% content_for :card do %>
   <%= render @return_status %>
 
-  <% unless @error&.auto_wait %>
+  <% unless @error&.auto_wait && app_time.before?(Rails.configuration.state_file_end_of_in_progress_intakes) %>
```

Comment:
`if !@error&.auto_wait || app_time.after?(Rails.configuration.state_file_end_of_in_progress_intakes)`

would make it easier to read.

Reply:
Done